indigo/indigo/src/main/scala/indigo/shared/events/GlobalEvent.scala

```diff
@@ -91,11 +91,12 @@ case object FullScreenExitError extends ViewEvent
   * behavior of Scala 3 enums to match the button number
   */
 enum MouseButton derives CanEqual:
-  case LeftMouseButton      extends MouseButton
-  case MiddleMouseButton    extends MouseButton
-  case RightMouseButton     extends MouseButton
-  case BrowserBackButton    extends MouseButton
-  case BrowserForwardButton extends MouseButton
+  case LeftMouseButton, MiddleMouseButton, RightMouseButton, BrowserBackButton, BrowserForwardButton
+
+/** Represents in which direction the mouse wheel was rotated
+ */
+enum MouseWheelDir derives CanEqual:
+  case Up, Down
```

Comment:
In general, I avoid abbreviations. Here I would probably prefer either `MouseWheel.Up` or maybe better(?) `MouseWheel.ScrollUp`.